leshan-client-core/pom.xml

```diff
@@ -34,6 +35,12 @@ Contributors:
             <groupId>org.eclipse.leshan</groupId>
             <artifactId>leshan-core</artifactId>
         </dependency>
+	<!-- Dependency for OSCORE support -->
+        <dependency>
+                <groupId>org.eclipse.californium</groupId>
+                <artifactId>cf-oscore</artifactId>
+                <version>${californium.version}</version>
+        </dependency>
```

Comment:
`leshan-client-code` must not depend to californium. (see https://github.com/eclipse/leshan/wiki/Modules#leshan-client-core)

Reply:
I see, thanks for the link. I will amend the code so leshan-client-core does not have any dependencies from Californium. I can probably keep it as an int instead of AlgorithmID.

Reply:
This should now be taken care of in commit 8fc7bfcdb67d219f712c5f2a1536a2d4c692e860
I needed the hex2ByteArray method from StringUtil in Californium. For now I extracted and took the code from there. If that is not fine I can re-implement it.

Reply:
maybe you could find what you need in `org.eclipse.leshan.util.Hex` ?

Reply:
> maybe you could find what you need in `org.eclipse.leshan.util.Hex` ?

Yes, I will check that.

Reply:
Update to use a method from the Leshan Hex class rather than the imported method from Californium in commit ebc728a69129645fe1347e312c1feb34ee4aae13